x-pack/plugins/task_manager/server/task_claimers/strategy_update_by_query.ts

```diff
@@ -230,7 +224,7 @@ function accumulateClaimOwnershipResults(
         tasksConflicted: stats.tasksConflicted + prev.stats.tasksConflicted,
         tasksClaimed: stats.tasksClaimed + prev.stats.tasksClaimed,
       },
-      docs,
+      docs: [...prev.docs, ...docs],
```

Comment:
Yea I was a little confused by it as well. I think likely we didn't run into issues very often because we don't have many limited concurrency batches (just reporting I think?)